src/main/java/com/wwi21sebgroup5/cinema/services/LoginService.java

```diff
@@ -32,7 +36,7 @@ public class LoginService {
      * @throws EmailAlreadyExistsException Thrown if the email already exists
      */
     public User register(RegistrationRequestObject registrationObject) throws UserAlreadyExistsException,
-            EmailAlreadyExistsException, CityNotFoundException {
+            EmailAlreadyExistsException {
         Optional<User> foundUser = userService.getUserByUserName(registrationObject.getUserName());
 
         if (foundUser.isEmpty()) {
```

Comment:
Auch wenn es in diesem Pull request nicht darum geht, aber es ist mir gerade aufgefallen...
Wieso wird hier nicht direkt eine UserAlreadyExistsException geworfen? Ist die Absicht dahinter, das die Email vorrangig behandelt wird? Wenn das s ist, wieso ist die Abfrage geschachtelt? Man könnte ja auch erst nach der Mail und dann nach dem Namen suchen oder?